module-2/backend/week_one.md

```diff
@@ -5,28 +5,57 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get: Get is used to access a page for read purposes
+  Post: Post is used for creating new data within an app
+  Delete: Delete is used to delete data within an app
+  Patch: Patch updates part of a resource, but not the whole thing. 
+  Put: updates a whole resource. 
 2. What is Sinatra?
+  Sinatra is a framework for making applications that incorporates Active Record and SQL databases. 
 4. What is MVC?
+  MVC stands for the Mode-View-Controller model for structuring applications
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  We use the RESTful route structure so that our applications are easy to read and understand according to a standardized structure.  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Global Variables

```

Comment:
instance variables (vs. local) was what we were looking for but global works too, i guess!